src/main/java/cloud/fogbow/ras/core/datastore/services/FogbowDatabaseService.java

```diff
@@ -0,0 +1,77 @@
+package cloud.fogbow.ras.core.datastore.services;
+
+import cloud.fogbow.common.exceptions.UnexpectedException;
+import cloud.fogbow.ras.core.datastore.StorableObjectTruncateHelper;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.transaction.TransactionSystemException;
+
+import javax.persistence.RollbackException;
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import java.util.Set;
+
+// TODO We might want to move this class to the Common project
```

Comment:
Act on this TODO. The class should go to common if it is used by multiple services and if it does not import packages from any particular service. Currently this is not the case. Would it be possible to move cloud.fogbow.ras.core.datastore.StorableObjectTruncateHelper to common, as well? If not, then this class cannot be moved to common as it is.

Reply:
The only problem on moving this class to common is that we would need to add a few dependencies to common.